ruby/basic_ruby/input_and_output.md

```diff
@@ -80,21 +83,24 @@ This is a sentence.
 ```
 
 ### Assignment
+
 <div class="lesson-content__panel" markdown="1">
 
 1. Read through the [Hello, World](https://www.learnrubyonline.org/en/Hello_World) lesson and exercise from [www.learnrubyonline.org](https://www.learnrubyonline.org).
-2. Read the first four sections of the [Ruby - File I/0](https://www.tutorialspoint.com/ruby/ruby_input_output.htm) from TutorialsPoint on `puts`, `gets`, `putc`, and `print`.
-3. [Read this article from Ruby Guides](https://www.rubyguides.com/2018/10/puts-vs-print/) which describes the differences between `puts`, `print` and `p`.
+1. Read the first four sections of the [Ruby - File I/0](https://www.tutorialspoint.com/ruby/ruby_input_output.htm) from TutorialsPoint on `puts`, `gets`, `putc`, and `print`.
+1. Read this [article from Ruby Guides](https://www.rubyguides.com/2018/10/puts-vs-print/) which describes the differences between `puts`, `print` and `p`.
```

Comment:
```suggestion
1. Read this [Ruby guides article describing the differences between puts, print and p.](https://www.rubyguides.com/2018/10/puts-vs-print/) which describes the differences between `puts`, `print` and `p`.
```
When the linter complains about links, you can often get around it by just putting "this" or "here" outside the link. But the intent of the rule is to make the link text more accessible to people who may be browsing our content with assistive technologies.

So I'm suggesting this change to make it clear where the user is going and what is behind the link. Hopefully that makes sense.

Reply:
Perfect